packages/main/src/plugin/index.ts

```diff
@@ -306,6 +309,86 @@ export class PluginSystem {
       'feedback',
       undefined,
     );
+
+    if (import.meta.env.PROD) {
+      // Get the current version of our application
+      const currentVersion = `v${app.getVersion()}`;
+
+      // Add update entry to the status bar
+      // this will show the current version, toolbar for the version number as well.
+      statusBarRegistry.setEntry(
+        'version',
+        false,
+        0,
+        currentVersion,
+        `Using version ${currentVersion}`,
+        'fa-light fa-circle-info',
+        true,
+        'version',
+        undefined,
+      );
+
+      // Only import on production builds
+      import('electron-updater').then(({ autoUpdater }) => {
+        // Setting the feed URL is now required as we are now using
+        // autoUpdater.on('update-available', () => {
+        // in other parts of the code
+        autoUpdater.setFeedURL({
+          provider: UPDATER_PROVIDER,
+          owner: UPDATER_OWNER,
+          repo: UPDATER_REPO,
+        });
+
+        // autoUpdater.checkForUpdatesAndNotify() is called from main/src/index.ts
+        autoUpdater.on('update-available', () => {
+          // If there is an update available, remove the "version" entry from the status bar
+          // and add the "update" entry
+          statusBarRegistry.removeEntry('version');
+
+          // Will add the new entry.
+          // If there is already an entry available, it'll update the icon to the new one (if it's different)
+          statusBarRegistry.setEntry(
+            'update',
+            false,
+            0,
+            currentVersion,
+            'Update available',
+            UPDATER_UPDATE_AVAILABLE_ICON,
+            true,
+            'update',
+            undefined,
+          );
+        });
+
+        // Update will create the standard "autoUpdater" dialog / update process that Electron provides
+        // TODO: After UI/UX finalization / feedback, we can replace this with a custom dialog
+        commandRegistry.registerCommand('update', () => {
+          // Use checkForUpdates (not andNotify) which will return the update information
+          // for us to retrieve the version used
+          autoUpdater.checkForUpdates().then(update => {
+            // Get the version if autoUpdater was able to retrieve update (not undefined)
+            const updateVersion = update?.updateInfo.version ? `(v${update?.updateInfo.version}) ` : '';
+
+            // Show the dialog with the version number if available
+            dialog
+              .showMessageBox({
+                type: 'info',
+                title: 'Update Available',
+                message: `A new version ${updateVersion}of Podman Desktop is available. Do you want to update your current version (${currentVersion})?`,
```

Comment:
```suggestion
                message: `A new version ${updateVersion} of Podman Desktop is available. Do you want to update your current version (${currentVersion})?`,
```

Reply:
the space is intentional in case there is an issue getting updateVersion (see above other code)